core/domain/value_generators_domain.py

```diff
@@ -69,8 +69,8 @@ def get_html_template(cls) -> bytes:
     # are set to 'Any'.
     def generate_value(
         self,
-        *args: Tuple[Any],
-        **kwargs: Dict[str, Any]
+        *args: Any,
+        **kwargs: Any
```

Comment:
For `*args`, tuple is added by default, by this I mean:
if we do `*args: str` then its type is  Tuple[str] but if we do `*args: Tuple[str]` then its type is considered as `Tuple[Tuple[str], ...]`
Similar goes for `**kwargs`:
If we do `**kwargs: int` then its type is `Dict[str, int]` but if we do `**kwargs: Dict[str, int]` then its type is considered as `Dict[str, Dict[str, int]` 

Fo reference: (https://stackoverflow.com/a/37032111)